ghedesigner/radial_numerical_borehole.py

```diff
@@ -318,18 +284,7 @@ def calc_sts_g_functions(self, single_u_tube, final_time=None, calculate_at_bh_w
         time = 1e-12 - 120
         time_step = 120
 
-        # type_idx = self.cell_inputs["type"]
-        r_in_idx = self.cell_inputs["inner-radius"]
-        r_center_idx = self.cell_inputs["center-radius"]
-        r_out_idx = self.cell_inputs["outer-radius"]
-        # thickness_idx = self.cell_inputs["thickness"]
-        k_idx = self.cell_inputs["conductivity"]
-        rho_cp_idx = self.cell_inputs["heat-capacity"]
-        temperature_idx = self.cell_inputs["initial-temperature"]
-        previous_temperature_idx = self.cell_inputs["initial-temperature"]
-        volume_idx = self.cell_inputs["volume"]
```

Comment:
The indices of the various layers are not changing from timestep to timestep, so no need to look them up at each iteration. I elevated this up to the ctor level so it only happens once.